woocommerce/woocommerce
# Fix notice styles

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Closes #42690.





In p1702294257650489/1702031295.900509-slack-C02UBB1EPEF, @jimjasson reported that the notices are showing the old styles when adding a product to the cart. This was detected due to a failing e2e test that ran against the nightly build of Woo core.

Initially, we thought that this issue was caused by https://github.com/woocommerce/woocommerce-blocks/pull/12043. While investigating this issue, I noticed that while migrating WooCommerce Blocks into core, the folder with the notice templates had been renames from `notices` to `block-notices`. This change took place in [https://github.com/woocommerce/woocommerce/commit/6950115000d562f53271f719a7c2070ffce4c4f9#diff-04997489ff1d17d1c6f2a1[…]f40a1ee468ba2c6a58be46dab10](https://github.com/woocommerce/woocommerce/commit/6950115000d562f53271f719a7c2070ffce4c4f9#diff-04997489ff1d17d1c6f2a147418afe91041c0f40a1ee468ba2c6a58be46dab10). 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

#### Testing fixed notices

1. Install and activate the [Twenty Twenty-Four theme](https://wordpress.org/themes/twentytwentyfour/).
2. Create a test page and add the cart shortcode to it.
3. Go to the frontend and visit a product page.
4. Add the product to the cart.
5. Verify that the new notice style is visible.

<table>
<tr>
<td valign="top">Old notice style:
<br><br>
<img width="1279" alt="Screenshot 2023-12-12 at 16 33 09" src="https://github.com/woocommerce/woocommerce/assets/3323310/8442fd57-a2b4-4f56-89b7-2026663eaaca">
</td>
<td valign="top">New notice style:
<br><br>
<img width="1280" alt="Screenshot 2023-12-12 at 16 30 36" src="https://github.com/woocommerce/woocommerce/assets/3323310/a3eb406d-092e-4ce4-82f7-5b633971ab38">
</td>
</tr>
</table>

6. Go to the test page with the cart shortcode.
7. Empty all form fields and submit the checkout form.
8. Verify that the new notice style is visible.

<table>
<tr>
<td valign="top">Old notice style:
<br><br>
<img width="1352" alt="Screenshot 2023-12-15 at 13 38 03" src="https://github.com/woocommerce/woocommerce/assets/3323310/386d46aa-2835-4e7d-8164-0d0136ae8af7">
</td>
<td valign="top">New notice style:
<br><br>
<img width="1352" alt="Screenshot 2023-12-15 at 13 35 47" src="https://github.com/woocommerce/woocommerce/assets/3323310/de55a668-9bd5-468f-bc36-426b6ecf89e4">
</td>
</tr>
</table>

> [!IMPORTANT]
> Please note that when using the Storefront theme, the SVG icon before the notice text is not visible. This issue should be addressed in a separate PR as it only affects the Storefront theme.

#### Testing existing notices

1. Go to `/my-account/`.
2. Log out, if you're currently logged in.
3. Simulate a login, but provide an incorrect username or password.
4. Verify that the error message is still working, and using the new style.
6. Log in and go to `/my-account/edit-address/`.
10. Edit the billing or shipping address.
11. Verify that the success message is still working, and using the new style.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Fix: Ensure that new notice styles are visible.

#### Comment 

</details>
